### DISCUSSION TITLE
Reset to default App Setting button

### DISCUSSION THREAD

**Discussion by trySometngNew** [SimonHalvdansson/Harmonic-HN#105] [Ideas]
Hi @SimonHalvdansson,
Can you create a revert to default Harmonic app settings button in settings menu?
In case user wants to default to original default Harmonic app settings to revert back their custom setting values in case they are unsure what setting value change from their end changed app behavior in some manner. This is especially useful since there is no save button on settings page to confirm if user setting change is intended or unintended one.

Regards,
Sourabh

**Comment by SimonHalvdansson** [comment]
Actually very reasonable, I'm adding it as an issue
